app/controllers/api/motions_controller.rb

```diff
@@ -11,6 +11,22 @@ def close
     respond_with_resource
   end
 
+  def create_outcome
+    load_and_authorize(:motion, :create_outcome)
+    MotionService.create_outcome(motion: @motion,
+                                 params: permitted_params.motion,
+                                 actor:  current_user)
+    respond_with_resource
+  end
+
+  def update_outcome
+    load_and_authorize(:motion, :update_outcome)
```

Comment:
Ah right. I suppose we'll go back to just loading it there for now, although it's a little confusing to separate loading and authorizing like that.. I totally agree about needing to auth in the service though.